scripts/manual_operations/tests/test_manual_remove.py

```diff
@@ -208,16 +205,14 @@ def test_main(self, mock_delete, mock_process, mock_find):
     @patch('scripts.manual_operations.manual_remove.ManualRemove.find_runs_in_database')
     @patch('scripts.manual_operations.manual_remove.ManualRemove.process_results')
     @patch('scripts.manual_operations.manual_remove.ManualRemove.delete_records')
-    @patch('scripts.manual_operations.manual_remove.handle_input')
-    def test_main_range(self, mock_handle_input, mock_delete, mock_process, mock_find):
+    @patch('scripts.manual_operations.manual_remove.user_input_check')
+    def test_main_range(self, mock_delete, mock_process, mock_find, mock_uic):
```

Comment:
It's a bit of a subtlety of python here where the patched method parameters are read in as arguments from bottom to top so mock_uic should be placed just after `self` such as `def test_main_range(self, mock_uic, mock_delete, mock_process, mock_find):`
Right now, mock_uic is representing `@patch('scripts.manual_operations.manual_remove.ManualRemove.find_runs_in_database')`